Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-breadcrumbs/ContestBreadcrumbs.tsx

```diff
@@ -16,7 +16,7 @@ const ContestBreadcrumbs = () => {
     const { themeColors, getColorClassName } = useTheme();
     const { breadcrumbItems } = useAppSelector((state) => state.contests);
 
-    const textColorClassName = getColorClassName(themeColors.textColor);
+    const textColorClassName = `getColorClassName(themeColors.textColor)`;
```

Comment:
we can use the textColor class name passed to the className of the object and have those quotes there, in example: className={`${textColorClassName} ${classname2}...`}

Reply:
Don't really know how this change happened. I'll revert it back